src/commands/batch.py

```diff
@@ -94,6 +95,7 @@ def view(batch_id, node):
             ['Batch', job.batch.id],
             ['Node', job.node],
             ['Command', job.batch.__name__()],
+            ['Arguments', job.batch.arguments],
             ['Exit Code', job.exit_code],
```

Comment:
Now that this table has been expanded to include the `Arguments`, it is pushing the `Exit Code` down the table. Realistically, admins looking over the jobs only care about the ones that failed.

We should probably move the `Exit Code` to be third row (or column?). The `Batch(id)` and `Node` should stay together as they form a key to the db table.

Reply:
This also keeps the `Command` and the `Arguments` together. The `Arguments` shouldn't get to long, hopefully.